tests/build/chains.go

```diff
@@ -75,7 +76,7 @@ var _ = framework.ChainsSuiteDescribe("Tekton Chains E2E tests", Label("ec", "HA
 			}
 
 			buildPipelineRunName = fmt.Sprintf("buildah-demo-%s", util.GenerateRandomString(10))
-			image = fmt.Sprintf("image-registry.openshift-image-registry.svc:5000/%s/%s", namespace, buildPipelineRunName)
+			image = fmt.Sprintf("quay.io/%s/test-images:/%s/%s", utils.GetQuayIOOrganization(), namespace, buildPipelineRunName)
```

Comment:
I doubt that a tag could start with "/"

Reply:
nice catch, thanks Cuiping!
Pavel has updated this, please check again.